tests/test_multitask_datamodule.py

```diff
@@ -257,9 +257,9 @@ def test_multitask_with_missing_fromsmiles_from_config_parquet(self):
         dm.setup()
 
         # self.assertEqual(len(dm), 100)                      # Should this have a fixed value for when it's initialized? MTL dataset only gets created after.
-        self.assertEqual(len(dm.train_ds), 602)  # type: ignore
-        self.assertEqual(len(dm.val_ds), 201)  # type: ignore
-        self.assertEqual(len(dm.test_ds), 201)  # type: ignore
+        self.assertEqual(len(dm.train_ds), 596)  # type: ignore
+        self.assertEqual(len(dm.val_ds), 199)  # type: ignore
+        self.assertEqual(len(dm.test_ds), 199)  # type: ignore
```

Comment:
Why do you change the length here? Are there molecules that fail the `found_size_mismatch` part? Which ones? Can we see the warning?